cmd/editaccount.go

```diff
@@ -818,6 +876,28 @@ func (p *EditAccountParams) Run(ctx ActionCtx) (store.Status, error) {
 		r.Add(s.Details...)
 	}
 
+	// if they provided the flag we change it
+	if flags.Changed("trace-context-subject") {
+		// if they set us to "", we disable it
+		if p.traceContextSubject == "" {
+			p.claim.Trace = nil
+			r.AddOK("disabled trace context")
+		} else {
+			p.claim.Trace = &jwt.MsgTrace{}
+			p.claim.Trace.Destination = jwt.Subject(p.traceContextSubject)
+			r.AddOK("changed trace context subject to %d", p.claim.Trace.Sampling)
```

Comment:
You are supposed to show the change of the subject, but you use `%d` and show the sampling value.